app/scripts/lib/setupAccountLabelsPetnamesBridge.ts

```diff
@@ -0,0 +1,90 @@
+import { NameController, NameType } from '@metamask/name-controller';
+import { cloneDeep } from 'lodash';
+import { CHAIN_IDS } from 'shared/constants/network';
+
+type IdentityEntry = {
+  address: string;
+  name: string;
+};
+
+type PertinentState = {
+  identities: { [address: string]: IdentityEntry };
+};
+
+type PertinentPreferencesController = {
+  store: {
+    getState: () => PertinentState;
+    subscribe: (callback: (state: PertinentState) => void) => void;
+  };
+};
+
+const ACCOUNT_LABEL_NAME_TYPE = NameType.ETHEREUM_ADDRESS;
+const ACCOUNT_LABEL_CHAIN_ID = CHAIN_IDS.MAINNET;
```

Comment:
What is the correct NameType and variation to use for the NameController::setName calls?